src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -573,7 +573,13 @@ class MavlinkStreamSysStatus : public MavlinkStream
 			bat_msg.energy_consumed = -1;
 			bat_msg.current_battery = (battery_status.connected) ? battery_status.current_filtered_a * 100 : -1;
 			bat_msg.battery_remaining = (battery_status.connected) ? ceilf(battery_status.remaining * 100.0f) : -1;
-			bat_msg.temperature = INT16_MAX;
+			bat_msg.temperature = (battery_status.connected) ? (int16_t)battery_status.temperature : INT16_MAX;
+			//bat_msg.average_current_battery = (battery_status.connected) ? battery_status.average_current_a * 100.0f : -1;
```

Comment:
Why is this commented?

Reply:
It needs an updated version of Mavlink with these messages to build correctly. I put them there for the future.